### PR TITLE
Add Brakeman, Rubocop and Coveralls to CI

### PR DESCRIPTION
**Pull request by klaseskilson**
~~Based on #140, merge that first.~~

This adds RuboCop and [Brakeman](https://github.com/presidentbeef/brakeman) to the Travis scripts. Also, Coveralls is added.

### CODE REVIEW

**Comment by klaseskilson** [comment]
With #140 merged, what do we think about this one?

File: .travis.yml [hunk 1]
```diff
@@ -1,11 +1,13 @@
 language: ruby
+sudo: false
```
Comment (danielronnkvist) [inline .travis.yml]: This should remove the warning from the travis logs right?

Comment (klaseskilson) [inline .travis.yml]: That was the idea, [but I don't think it's working](https://travis-ci.org/klaseskilson/vattentornet/builds/161130844#L122-L124).

Comment (danielronnkvist) [inline .travis.yml]: It should be `sudo: no`
https://docs.travis-ci.com/user/ci-environment/#Virtualization-environments

Comment (klaseskilson) [inline .travis.yml]: I'm not sure if they can be disabled at all?

**Comment by klaseskilson** [comment]
I'm merging this now. It's only development related except for the two new empty lines in the application helper.

**Pull request merged**